packages/components/src/components/carousel/carousel-nav.jsx

```diff
@@ -0,0 +1,60 @@
+import classNames from 'classnames';
+import PropTypes from 'prop-types';
+import React from 'react';
+import Icon from '../icon';
+
+const Nav = ({
+    active_index,
+    active_bullet_color,
+    bullet_color,
+    className,
+    handleNextClick,
+    handlePrevClick,
+    handleNavigationClick,
+    list,
+    show_bullet,
+    show_nav,
+}) => (
+    <nav className={classNames('dc-carousel__nav', className)}>
+        {show_nav && (
+            <span id='dt_contract_info_left_nav' className='dc-carousel__icon' onClick={handlePrevClick}>
+                <Icon icon='IcChevronLeft' />
+            </span>
+        )}
+        {show_bullet && (
+            <ul className='dc-carousel__nav-list'>
+                <li
+                    className={classNames('dc-carousel__nav-item', 'dc-carousel__nav-item--active')}
+                    style={{
+                        transform: `translate3d(${24 * active_index}px, 0, 0)`,
+                        backgroundColor: active_bullet_color,
+                    }}
```

Comment:
@mahdiaryayi-fs Can we style `li` element using `className` from parent instead of using inline style?